changelog/fix_should_not_offence_single_quoted_symbol_containing_double_quotes.md

```diff
@@ -0,0 +1 @@
+* [#11836](https://github.com/rubocop/rubocop/issues/11836): should not offense single-quoted symbol containing double quotes in `Lint/SymbolConversion`. ([@KessaPassa][])
```

Comment:
In order to pass the test, the changelog entry's sentence must start with an upper character.

```console
rspec './spec/project_spec.rb[1:3:4:13:3:4:1]' # RuboCop Project Changelog future entries For D:/a/rubocop/rubocop/changelog/fix_should_not_offence_single_quoted_symbol_containing_double_quotes.md entry body does not start with a lower case
```

Reply:
Thank you for your advice!
I'm checking if the "bundle exec rake ascii_spec" command passes in the local environment.

Reply:
Yeah, for example, you may reproduce the minimal test results by running the following:

```console
$ bundle exec rspec './spec/project_spec.rb[1:3:4:13:3:4:1]'
```

Reply:
Thank you!
I passed the rspec test.